### ISSUE TITLE
Deleting files makes git lfs status die with no such file or directory

### ISSUE THREAD

**Issue by abentley** [git-lfs/git-lfs#3464]
Steps to reproduce:
```
$ git --version
git version 2.17.1
$ git lfs --version
git-lfs/2.6.1 (GitHub; linux amd64; go 1.11.1)

$ git init enoent
Initialized empty Git repository in /home/abentley/sandbox/enoent/.git/
$ cd enoent
$ git lfs install
Updated git hooks.
Git LFS initialized.
$ echo foo > bar
$ git add bar
$ git commit -m "Added bar"
[master (root-commit) 7a3c898] Added bar
 1 file changed, 1 insertion(+)
 create mode 100644 bar
$ rm bar
$ git add bar
$ git lfs status
On branch master

Git LFS objects to be committed:

open /home/abentley/sandbox/enoent/bar: no such file or directory
```

**Comment by bk2204** [comment]
Hey, thanks for writing in.

I can confirm this behavior and it's definitely a bug. We should be handling the deleted file case, and we're not. I'll try to look into getting this fixed sometime soon.

**Comment by hivearts** [comment]
I have a lot of files deleted from my repository, and it can't push. How to fix that?

**Comment by bk2204** [comment]
@hivearts I think that's an issue that's distinct from this one. Could you open a new issue and describe the output you're seeing and the output of `git lfs env`, please?

**Comment by yanivbm** [comment]
I think had the same problem, 
I solved it by using the following command:
git lfs fetch --all <remote>

**Comment by bk2204** [comment]
Hey, I think this has been fixed in #3482, so I'm closing this. Thanks for reporting it.
